pkgs/top-level/all-packages.nix

```
@@ -14347,6 +14347,8 @@ in
 
   morty = callPackage ../servers/web-apps/morty { };
 
+  mullvad = callPackage ../applications/networking/mullvad { };
```

```suggestion
  mullvad-vpn = callPackage ../applications/networking/mullvad-vpn { };
```

You'll need to rename the actual directory as well.